h/static/scripts/tests/controllers/search-bar-controller-test.js

```diff
@@ -83,6 +92,82 @@ describe('SearchBarController', function () {
         });
     });
 
+    it('it shows suggestions for tags when facet is selected', function (done) {
```

Comment:
Hmm... we're testing several behaviors in a single test case here. We generally try to only check one behavior in a particular `it()` test, which tends to make debugging easier when tests fail. Where several test cases share common setup, that can be grouped into `beforeEach()` within a nested `describe()` block.